cmake/modules/AddSwift.cmake

```diff
@@ -348,7 +348,7 @@ function(_add_variant_link_flags)
   if("${LFLAGS_SDK}" STREQUAL "LINUX")
     list(APPEND result "-lpthread" "-latomic" "-ldl")
   elseif("${LFLAGS_SDK}" STREQUAL "FREEBSD")
-    list(APPEND result "-lpthread")
+    list(APPEND result "-lpthread" "-L/usr/local/lib" "-lobjc")
```

Comment:
I don't think this is quite right. We shouldn't really link objC on platforms where interoperability is disabled.

Reply:
Yeah, i'm trying to remove this dependency, couldn't solve this with any know flags/options, the only working solution i found is to completely avoid `.mm`. Couldn't we use `-lobjc` as a temporary fix? Isn't it required on any freebsd platform using the current build environment? Sorry, i'm not familiar with FreeBSD, just having fun 😊

Reply:
I think the correct solution is that of not emitting `__gnustep_objcxx_personality_v0` altogether. FWIW, I think clang is the right place to look at (but @jrose-apple / @compnerd probably know better). FWIW, I and @compnerd discussed about enabling obj-C interop on other platforms yesterday. Maybe we can consider taking a look at that on FreeBSD.

Reply:
We should get a working non-ObjC build working on FreeBSD first. Getting ObjC interop on a non-Apple platform will be complicated.